### PR TITLE
Update 2201.5.0 release note with OpenAPI changes

### PR DESCRIPTION
**Pull request by aneeshafedo**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 1]
```diff
@@ -161,6 +161,9 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 #### Language Server
 
 #### OpenAPI Tool
+- Updated the tool to handle multiple media types with same return code in the Ballerina service to OpenAPI contract generation.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:164]:
```suggestion
- Updated the tool to handle multiple media types with the same return code in the Ballerina service to OpenAPI contract generation. 
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md [hunk 2]
```diff
@@ -161,6 +161,9 @@ To view bug fixes, see the [GitHub milestone for 2201.5.0 (Swan Lake)](https://g
 #### Language Server
 
 #### OpenAPI Tool
+- Updated the tool to handle multiple media types with same return code in the Ballerina service to OpenAPI contract generation. 
+- Added the support to handle the `ballerina/http` module's newly introduced `@http:Query` annotation in Ballerina service to OpenAPI contract generation.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md:165]:
```suggestion
- Added the support to handle the newly introduced `@http:Query` annotation of the `ballerina/http` module in the Ballerina service to OpenAPI contract generation.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md (hunk 1 above)
Comment (aneeshafedo) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md]: Updated. Thank you

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md (hunk 2 above)
Comment (aneeshafedo) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md]: Updated. Thank you

**Review by praneesha — APPROVED** [review]

**Pull request merged**